test/components/canBeCreatedWithItems.ts

```diff
@@ -0,0 +1,60 @@
+import { IPokemon } from "../../src/components/Battles";
+import { INewPokemon } from "../../src/components/constants/Pokemon";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+const fsp: FullScreenPokemon = stubBlankGame();
```

Comment:
Create a new `fsp` inside each test. Although it's slower, it gives you a safer playground to work in. We don't want one test failing to leave some shared `fsp` in a bad state for subsequent tests.